tests/baselines/reference/emptyArrayBindingPatternParameter04.js

```diff
@@ -0,0 +1,11 @@
+//// [emptyArrayBindingPatternParameter04.ts]
+
+
+function f([] = [1,2,3,4]) {
```

Comment:
@mhegazy  I was thinking it would just emit

```JavaScript
function f(_a) {
    void [1,2,3,4];
}
```

I use `void` here to keep object literals from being read as block bodies, even though I don't think this could be an issue.